src/pkg/errs/errs_test.go

```diff
@@ -19,6 +20,23 @@ func TestErrUnitSuite(t *testing.T) {
 	suite.Run(t, &ErrUnitSuite{Suite: tester.NewUnitSuite(t)})
 }
 
+func (suite *ErrUnitSuite) TestInternal() {
+	table := []struct {
+		get  errEnum
+		errs []error
+	}{
+		{RepoAlreadyExists, []error{repository.ErrorRepoAlreadyExists}},
+		{BackupNotFound, []error{repository.ErrorBackupNotFound}},
+		{ServiceNotEnabled, []error{graph.ErrServiceNotEnabled}},
+		{ResourceOwnerNotFound, []error{graph.ErrResourceOwnerNotFound}},
+	}
+	for _, test := range table {
+		suite.Run(string(test.get), func() {
+			assert.Equal(suite.T(), test.errs, Internal(test.get))
```

Comment:
ElementsMatch is more safe here, since it evaluates to true independent of order.
```suggestion
			assert.ElementsMatch(suite.T(), test.errs, Internal(test.get))
```